src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

Тут ты используешь raw use параметризированного интерфейса MachineProducer, давай параметризируем его соответственным типом(и так для каждого Producer)